test/FilesTests/RavenFilesTestBase.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -----------------------------------------------------------------------
```

Comment:
We'll need to merge this with the `FastTests` (for now) code.
We don't want to have too many test proejcts.

For now it is fine while you are just looking at those, but we won't keep it long term